### PR TITLE
🌟feature(windows terminal): supported

### PR DESCRIPTION
**Pull request by cutbypham**
https://github.com/mcchrish/zenbones.nvim/pull/68#issuecomment-1003226246

### CODE REVIEW

**Comment by rktjmp** [comment]
:+1: Should `name` included in `check_keys`? 

https://docs.microsoft.com/en-us/windows/terminal/customize-settings/color-schemes#creating-your-own-color-scheme Doesn't really say what's required, but maybe? Most (all?) on https://windowsterminalthemes.dev/ have the `name` key.

**Comment by cutbypham** [comment]
> +1 Should `name` included in `check_keys`?
> 
> https://docs.microsoft.com/en-us/windows/terminal/customize-settings/color-schemes#creating-your-own-color-scheme Doesn't really say what's required, but maybe? Most (all?) on https://windowsterminalthemes.dev/ have the `name` key.
 
- yep :+1: , i just add the `name` in `check_keys`,  will it work?

File: lua/shipwright/transform/contrib/windows_terminal.lua [hunk 1]
```diff
@@ -0,0 +1,85 @@
+--- windows terminal transform, expects a table in the shape:
+--
+-- @param colors {
+-- "name" : "Theme Name",
+
+-- "cursorColor": "#000000",
+-- "selectionBackground": "#000000",
```
Comment (rktjmp) [inline lua/shipwright/transform/contrib/windows_terminal.lua]: Need to be `cursor_color` and `selection_background`, those are the actual keys used in the template substitution.

File: lua/shipwright/transform/contrib/windows_terminal.lua [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+-- "white" : "#000000",
+-- "yellow" : "#000000",
+-- "brightBlack" : "#000000",
+-- "brightBlue" : "#000000",
+-- "brightCyan" : "#000000",
+-- "brightGreen" : "#000000",
+-- "brightPurple" : "#000000",
+-- "brightRed" : "#000000",
+-- "brightWhite" : "#000000",
+-- "brightYellow" : "#000000"
```
Comment (rktjmp) [inline lua/shipwright/transform/contrib/windows_terminal.lua]: Need to be `bright_black`, etc, those are the actual keys used in the template substitution.

File: lua/shipwright/transform/contrib/windows_terminal.lua [hunk 3]
```diff
@@ -0,0 +1,85 @@
+--- windows terminal transform, expects a table in the shape:
+--
+-- @param colors {
+-- "name" : "Theme Name",
```
Comment (rktjmp) [inline lua/shipwright/transform/contrib/windows_terminal.lua]: This  `@param` doc should be a "lua table", so it should instead be `name = "Theme Name",`, etc for all the keys.

**Review by rktjmp — COMMENTED** [review]
Just need to correct the `@param` documentation, it should be a valid lua table, see comments. Sorry, didn't notice it before.

**Comment by rktjmp** [comment]
:anchor: :+1:

**Pull request merged**